docker-compose.yaml

```diff
@@ -30,6 +30,9 @@ services:
             - camunda-cloud
         depends_on:
             - elasticsearch
+        profiles:
+            - webapps
+            - full
```

Comment:
These profiles should be documented. Are both of them needed?

Reply:
Currently, both profiles `webapps` and `full` are the same. We could drop the `webapps` profile until it is needed.